src/components/Calculator.jsx

```diff
@@ -1,77 +1,60 @@
-    return (
-      <div>
-        <div className="screen">{display}</div>
-        <div className="row">
-          <Key symbol="AC" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="+/-" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="%" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="÷" color="orange" retrieveSymbol={this.retrieveSymbol} />
-        </div>
-        <div className="row">
-          <Key symbol="7" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="8" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="9" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="x" color="orange" retrieveSymbol={this.retrieveSymbol} />
-        </div>
-        <div className="row">
-          <Key symbol="4" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="5" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="6" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="-" color="orange" retrieveSymbol={this.retrieveSymbol} />
-        </div>
-        <div className="row">
-          <Key symbol="1" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="2" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="3" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="+" color="orange" retrieveSymbol={this.retrieveSymbol} />
-        </div>
-        <div className="row">
-          <Key symbol="0" color="double" retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="." retrieveSymbol={this.retrieveSymbol} />
-          <Key symbol="=" color="orange" retrieveSymbol={this.retrieveSymbol} />
-        </div>
+  return (
+    <div>
+      <div className="screen">{display}</div>
+      <div className="row">
+        <Key symbol="AC" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="+/-" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="%" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="÷" color="orange" retrieveSymbol={retrieveSymbol} />
+      </div>
+      <div className="row">
+        <Key symbol="7" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="8" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="9" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="x" color="orange" retrieveSymbol={retrieveSymbol} />
+      </div>
+      <div className="row">
+        <Key symbol="4" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="5" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="6" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="-" color="orange" retrieveSymbol={retrieveSymbol} />
+      </div>
+      <div className="row">
+        <Key symbol="1" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="2" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="3" retrieveSymbol={retrieveSymbol} />
+        <Key symbol="+" color="orange" retrieveSymbol={retrieveSymbol} />
+      </div>
+      <div className="row">
```

Comment:
- [ ] OPTIONAL: This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: